geode-docs/tools_modules/gfsh/command-pages/create.html.md.erb

```diff
@@ -437,6 +451,13 @@ create gateway-receiver [--groups=value(,value)*] [--members=value(,value)*]
 <td>The host name or IP address told to gateway senders as the address for them to connect to. The locator informs gateway senders of this value.</td>
 <td> </td>
 </tr>
+<tr class="odd">
+<td><span class="keyword parmname">\-\-if-not-exists</span></td>
+<td>When specified without providing a boolean value or when specified and set to true, gateway receivers will not be created if they already exist. When the default value of false, an attempt to create a second gateway receiver is reported as an error.
```

Comment:
Since there can be only one gateway receiver per member, it might be more accurate to say that `--if-not-exists` permits execution to ignore failure due to a member still existing.  That error is still reported and displayed, but is displayed as `SKIPPED: Gateway receiver already exists` rather than `ERROR: Gateway receiver already exists`.

Reply:
To take PurelyApplied's comment further, it would be even more accurate to say that `--if-not-exists=true` (or `--if-not-exists` without specifying a boolean value) will report members that already have a gateway receiver without reporting it as an error.

Reply:
I have revised the --if-not-exists defn to not mention error (since the command succeeds), instead stating that the output lists the status of each GW receiver creation attempt.

@PurelyApplied @pdxrunner Please review (again).